cairo-libs/packages/cometbft/src/utils.cairo

```diff
@@ -33,3 +33,54 @@ impl FractionAsProtoName of ProtoName<Fraction> {
         "Fraction"
     }
 }
+
+
+pub impl SpanU8TryIntoU256 of TryInto<Span<u8>, u256> {
+    /// Decodes as big endian.
+    fn try_into(self: Span<u8>) -> Option<u256> {
+        // Only allows size 32 to ensure all bytes fit exactly into a `u256`.
+        if (self.len() != 32) {
+            return Option::None;
+        }
+
+        const N256_POW_1: u128 = 0x100;
```

Comment:
it can be just `N256` (I used `N` as _number_, as the variable names can't start with numbers) or `U8_MAX`.

Reply:
[b5f3f21](https://github.com/informalsystems/ibc-starknet/pull/361/commits/b5f3f21c49957c9aaa78f66db8fe10f7ab6a65df)